github/EnterpriseConsumedLicenses.pyi

```diff
@@ -0,0 +1,23 @@
+from github.GithubObject import CompletableGithubObject
```

Comment:
Please embed typing of new files in the `py` file. For examples, see https://github.com/PyGithub/PyGithub/commit/8bdfc3a0d7fc7391ff34e60e20a6987433025417

Reply:
Sure, will embed it in the `py` file without the `pyi` file.